docs/machine-learning/how-to-guides/consuming-model-ml-net.md

```diff
@@ -7,6 +7,11 @@ ms.custom: mvc,how-to
 ---
 # Operationalize a trained machine learning model in apps - ML.NET
 
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+
+This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
```

Comment:
Added the dot at the end of the paragraph:
```suggestion
This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes).
```
@luisquintanilla this applicable to all other additions in this PR as well.

Reply:
Good catch @pkulikov . Thanks.